frontend/src/Settings/Indexers/Options/IndexerOptions.js

```diff
@@ -85,7 +85,7 @@ function IndexerOptions(props) {
                 type={inputTypes.CHECK}
                 name="preferIndexerFlags"
                 helpText={translate('PreferIndexerFlagsHelpText')}
-                helpLink="https://wiki.servarr.com/Definitions#Indexer_Flags"
+                helpLink="https://wiki.servarr.com/en/radarr/settings#options"
```

Comment:
drop the `en`

and use `https://wiki.servarr.com/radarr/settings#indexer-flags`  I just updated the wiki for that